src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Mapping
+
+from sentry.sentry_metrics.configuration import UseCaseKey
+
+
+class UseCaseID(Enum):
+    PERFORMANCE = "performance"
+    RELEASE_HEALTH = "release-health"
```

Comment:
The MRI namespaces used for these cases today are transactions and sessions, what's the reason for changing the names ?

Reply:
This `UseCaseID` is what is ultimately populating the `use_case_id` field in messages' payloads. It's also what Clickhouse knows to be the `use_case_id`. Is there a reason why we should change these from performance and release-health to transactions and sessions? (Also to be clear, we are not changing the namespaces. We are just introducing the `UseCaseID` enum to keep track of the "true" use cases that are identifying messages and that Clickhouse recognizes ultimately. We also have a mapping between `UseCaseID` and namespace)

Reply:
I think the true use cases should be `transactions` and `sessions` not `performance` and `release_health` so you avoid the mapping altogether. 
I don't think we are storing anything as use case in Clickhouse today, so you would not be changing the existing value.

Another reason for not doing the pamming. Let's say you map what comes from MRI (`transactions`) to what the indexer consider the true use case (`performance`). What if somebody tomorow introduces a new MRI namespace called `performance` in the intent of creating a new use case? Now you would have a naming clash.

I'd just take transactions and sessions as use case ids and avoid the mapping.